internal/testrunner/runners/system/runner.go

```diff
@@ -747,6 +752,17 @@ func validateFields(docs []common.MapStr, fieldsValidator *fields.Validator, dat
 	return nil
 }
 
+func assertHitCount(expected int, docs []common.MapStr) (pass bool, message string) {
+	observed := len(docs)
```

Comment:
The variable should be declared as close to where it is used as possible to minimize scope. So this can be moved within the `if` block.

Reply:
applied